vm-systemd/qubes-iptables.service

```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=Qubes base firewall settings
+Requires=qubes-iptables.service
+After=qubes-iptables.service
```

Comment:
Uhm, now it references itself...

But also, I think the `qubes-antispoof.service` should be included in the preset file too, so it would be started even if somebody disables `qubes-iptables.service` (via systemctl, not qvm-service), as `qubes-iptables.service` shouldn't really be necessary for proper `vif-route-qubes` function.